src/components/Camera/style.css

```diff
@@ -1,9 +1,28 @@
+.camera {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
 .video-overlay {
   /* Relatively positioned so absolute children are bound to this overlay */
   position: relative;
   overflow: hidden;
-  margin-top: 16px;
-  width: 100%;
+  margin: 0 40px 16px;
+}
+
+.fullScreen {
```

Comment:
is full screen layouting done at step level too? can it not be done at container level only?